tests/e2e/resources/installation_config/cognito-rds-s3-static.yaml

```diff
@@ -318,6 +318,17 @@ profiles-and-kfam:
         - key: kustomize.component
           value: profiles
 
+
+#user namespace
+user-namespace:
+  installation_options:
+    kustomize:
+      paths: 
+        - ../../awsconfigs/common/user-namespace/overlay
```

Comment:
static one should point to upstream not the aws overlay. same for cognito

Can you also update the Cognito guide to create a user with email user@example.com? https://awslabs.github.io/kubeflow-manifests/docs/deployment/cognito/manifest/guide/#60-connect-to-the-central-dashboard